app/src/main/res/values/dimens.xml

```diff
@@ -399,4 +399,20 @@
   <!-- Previous Responses Item: Question View -->
   <dimen name="previous_responses_item_question_view_margin_start">44dp</dimen>
   <dimen name="previous_responses_item_question_view_margin_end">44dp</dimen>
+
+  <!-- General Button Item: Exploration Split View -->
```

Comment:
https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/7d089aae-3288-4474-8e4b-14a6f67878f3/specs/

Reply:
The same applies to here, this is the mobile res directory so we might not be using the split resources here.

Reply:
Actually Sliptview depends on two things: (a.) Physical size of more than 7" and (b.) DragAndDrop/ImageRegionSelection interaction

It is independent of tablet or mobile or its configuration

So there might come a mobile which is more than 7" and in that case will need to use split screen.